src/index.html

```diff
@@ -6,9 +6,36 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#" class="logo">
+        <img src="images/logo.png" alt="MOYO">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
```

Comment:
```suggestion
            <a href="#apple" class="nav__link is-active">
               Apple
            </a>
```
![Screenshot 2023-02-23 at 17 38 31](https://user-images.githubusercontent.com/6526149/220955908-263fece1-21f2-4a0c-b635-312ff559801f.png)